docs/compose/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+locale: en
+slug: /compose
+contentful_id: 2SZT4NhoGRHZLVY01b7HXJ
```

Comment:
Remove contentful_id from all compose pages (because they are not in Contentful)